ansible/playbooks/teleop-deploy.yml

```diff
@@ -1,8 +1,10 @@
 ---
-- name: Add IP address of all hosts to server (running Ansible)
-  hosts: server
+- name: Add IP address of all hosts to computer that is running Ansible
```

Comment:
have you merged master to your branch? this is showing a "change" but it's already in master